packages/runner/lib/app.ts

```diff
@@ -2,6 +2,9 @@ import './tracer.js';
 import { server } from './server.js';
 import { stringifyError } from '@nangohq/utils';
 import { logger } from './utils.js';
+import { launchProvidersSync } from '@nangohq/shared';
+
+await launchProvidersSync();
```

Comment:
`sync` is a loaded word within the context of Nango so I would advise another name for this method. Maybe something with `monitor`?